docker/default.yml

```diff
@@ -12,7 +12,7 @@ plugin:
 inbound-transport:
   - [http, 0.0.0.0, 3000]
   - [ws, 0.0.0.0, 3002]
-outbound-transport: http
+outbound-transport: ws
```

Comment:
Did we want this change? I think we typically want http as our default outbound transport (if an open ws is made over inbound, that open socket will be used for outbound as well)

Reply:
Ah beans, I forgot about that. I'll nix it when I'm back in front of my computer